test/buy-tshirt.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { $, browser } from 'protractor';
+
+describe('When I will buy a t-shirt', () => {
+
+  beforeEach(() => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
+  });
+
+  it('Then should be bought a t-shirt', async () => {
```

Comment:
mejor `Then a t-shirt should be bought`

Reply:
@sorregom, al parecer si se necesita, el error que tslint me sugiere es que esta declarado, pero nunca sera leído